### PR TITLE
feat(demo): update demo app with new features

### PR DESCRIPTION
**Pull request by goncalo-frade-iohk**
### Description: 
Update demo app with new UI.

### Checklist: 
- [x] My PR follows the [contribution guidelines](https://github.com/input-output-hk/atala-prism-wallet-sdk-swift/blob/main/CONTRIBUTING.md) of this project
- [x] My PR is free of third-party dependencies that don't comply with the [Allowlist](https://toc.hyperledger.org/governing-documents/allowed-third-party-license-policy.html#approved-licenses-for-allowlist)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked the PR title to follow the [conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0/)

### CODE REVIEW

File: AtalaPrismSDK/PrismAgent/Sources/PrismAgent+Backup.swift [hunk 1]
```diff
@@ -0,0 +1,356 @@
+import Domain
```
Comment (hamada147) [inline AtalaPrismSDK/PrismAgent/Sources/PrismAgent+Backup.swift:1]: Should this PR contains logic for the backup?!

File: Sample/AtalaPrismWalletDemo/AtalaPrismWalletDemo/Helper/AtalaSwiftUIComponents/CommonUI/ClosableSheet.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
+//
```
Comment (hamada147) [inline Sample/AtalaPrismWalletDemo/AtalaPrismWalletDemo/Helper/AtalaSwiftUIComponents/CommonUI/ClosableSheet.swift:1]: Remove copyright

File: Sample/AtalaPrismWalletDemo/AtalaPrismWalletDemo/Helper/AtalaSwiftUIComponents/CommonUI/HyperlinkText.swift [hunk 3]
```diff
@@ -0,0 +1,214 @@
+//
+//  HyperlinkText.swift
```
Comment (hamada147) [inline Sample/AtalaPrismWalletDemo/AtalaPrismWalletDemo/Helper/AtalaSwiftUIComponents/CommonUI/HyperlinkText.swift:2]: Remove copyright

File: Sample/AtalaPrismWalletDemo/AtalaPrismWalletDemo/Helper/AtalaSwiftUIComponents/CommonUI/WordTagEditable.swift [hunk 4]
```diff
@@ -0,0 +1,97 @@
+//
```
Comment (hamada147) [inline Sample/AtalaPrismWalletDemo/AtalaPrismWalletDemo/Helper/AtalaSwiftUIComponents/CommonUI/WordTagEditable.swift:1]: Remove copyright

File: AtalaPrismSDK/PrismAgent/Sources/PrismAgent+Backup.swift (hunk 1 above)
Comment (goncalo-frade-iohk) [reply AtalaPrismSDK/PrismAgent/Sources/PrismAgent+Backup.swift]: Yes it was a mistake already removed.

File: Sample/AtalaPrismWalletDemo/AtalaPrismWalletDemo/Helper/AtalaSwiftUIComponents/CommonUI/WordTagEditable.swift (hunk 4 above)
Comment (goncalo-frade-iohk) [reply Sample/AtalaPrismWalletDemo/AtalaPrismWalletDemo/Helper/AtalaSwiftUIComponents/CommonUI/WordTagEditable.swift]: Done also to all the above

**Review by elribonazo — APPROVED** [review]
LGTM!

**Comment by mkbreuningIOHK** [comment]
@goncalo-frade-iohk , just for clarification: in the title of the PR, it is mentioning 'a new feature' but in the description, it is not explicitly writing what's the new feature. Could you add this in the PR description? 
As from the [v6.0.0 release note](https://github.com/input-output-hk/atala-prism-wallet-sdk-swift/releases/tag/6.0.0), it seems to be linked to this new feature: "add jwt and anoncreds verification".

**Pull request merged**